src/shipit_static_analysis/shipit_static_analysis/workflow.py

```diff
@@ -80,19 +80,20 @@ def run(self, revision):
         logger.info('Modified files', files=modified_files)
 
         # mach configure
-        logger.info('Mach configure ...')
+        logger.info('Mach configure…')
```

Comment:
Fair enough, I will revert this change with the next rebase. You win again, legacy software!